### ISSUE TITLE
AZ Bicep version and decomplie failing for versions v0.3.256 and v0.3.126 -- same "fromisoformat" attribute error

### ISSUE THREAD

**Issue by briancjordan** [Azure/azure-cli#17694]
## Describe the bug
Running bicep version and decompile command fails.  Expect build to do the same but have not tried yet.

**Command Name**
`az bicep version`

**Errors:**
```
type object 'datetime.datetime' has no attribute 'fromisoformat'
Traceback (most recent call last):
cli\Lib\site-packages\knack\cli.py, ln 231, in invoke
azure\cli\core\commands\__init__.py, ln 657, in execute
azure\cli\core\commands\__init__.py, ln 720, in _run_jobs_serially
azure\cli\core\commands\__init__.py, ln 691, in _run_job
azure\cli\core\commands\__init__.py, ln 328, in __call__
site-packages\azure\cli\core\__init__.py, ln 807, in default_command_handler
azure\cli\command_modules\resource\custom.py, ln 3302, in show_bicep_cli_version
azure\cli\command_modules\resource\_bicep.py, ln 63, in run_bicep_command
azure\cli\command_modules\resource\_bicep.py, ln 152, in _load_bicep_version_check_result_from_cache
AttributeError: type object 'datetime.datetime' has no attribute 'fromisoformat'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

Execute commands as per help documentation.

- _Put any pre-requisite steps here..._
- `az bicep version`

## Expected Behavior

List current version of Bicep install.  In case of decompile, create Bicep file.

## Environment Summary
```
Windows-10-10.0.19041-SP0
Python 3.6.8
Installer: MSI

azure-cli 2.22.0

```

**Comment by yonzhan** [comment]
route to service team

**Comment by zhoxing-ms** [comment]
This issue is same as #17665, and we have fixed it in the hotfix version (`2.22.1`) of CLI
